Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -0,0 +1,74 @@
+// // SPDX-License-Identifier: Apache-2.0
+// // Licensed to the Ed-Fi Alliance under one or more agreements.
+// // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// // See the LICENSE and NOTICES files in the project root for more information.
+
+import { DeleteResult, Logger, DeleteRequest } from '@edfi/meadowlark-core';
+import { Client } from 'pg';
+import { deleteDocumentByIdSql } from './QueryHelper';
+
+// This function should only be used for testing, it will clear out all data from both documents and references tables
+export async function deleteAll(client: Client): Promise<DeleteResult> {
+  const deleteResult: DeleteResult = { response: 'UNKNOWN_FAILURE' };
+
+  try {
+    await client.query('BEGIN');
+    await client.query('DELETE FROM meadowlark.documents;');
+    await client.query('DELETE FROM meadowlark.references;');
+    await client.query('COMMIT');
```

Comment:
I think it's fine to have separate calls for each of these[[this is an example](https://github.com/brianc/node-postgres/issues/1252#issuecomment-293899088)](url) from the library creator, where he does something similar. But since this is a simple case with all the SQL together, I have no issue changing it to a single SQL statement

Reply:
@bradbanister any preference here? I don't actually have a strong feeling, if it does work as expected.